src/game.cpp

```diff
@@ -11322,6 +11330,23 @@ void game::takeoff(int pos)
     }
 }
 
+void game::change_side(int pos)
+{
+    if (pos == INT_MIN) {
+        auto filter = [this](const item &it) { return u.is_wearing_item(it); };
+        pos = inv_for_filter(_("Change side for item:"), filter);
+    }
```

Comment:
This block looks confusing for two reasons:
* It handles the case when `pos == INT_MIN` - is it ever called with such a position?
* When it called with such a position, it lists all worn items, including ones that cover both sides and thus changing their sides makes no sense

You could inline that filter and have it capture only `[&u]` instead of the pointer to `g`.